Assets/Mirror/Runtime/NetworkBehaviour.cs

```diff
@@ -350,7 +350,9 @@ protected bool SyncVarNetworkIdentityEqual(NetworkIdentity newIdentity, uint net
                 newNetId = newIdentity.NetId;
                 if (newNetId == 0)
```

Comment:
This check can be removed, since we should return true in case we want to check if newNetId == 0

Reply:
actually no.  because if the netid is 0,  it means the object has not been spawned yet.
There can be many objects that are not spawned, and they would all have netid ==0.

by returning false here,  we ensure that if we assign an object that is not spawned,  then we call
SetSyncVarNetworkIdentity. no matter what.

Reply:
Good point.